performance-tracking-core/src/main/java/com/rakuten/tech/mobile/perf/core/Metric.java

```diff
@@ -1,8 +1,16 @@
 package com.rakuten.tech.mobile.perf.core;
 
-public class Metric {
+class Metric {
 
-  static final long MAX_TIME = 10000L; // 10 s
+  private static long maxTime = 10000L; // 10 s
+
+  static void setMaxTime(long duration) {
+    maxTime = duration;
+  }
+
+  static long getMaxTime() {
```

Comment:
Can the function be called `maxTime` too - I think it reads a little better in usage.